src/Assignment-2/Program.cs

```diff
@@ -0,0 +1,422 @@
+                    }
+
+                case 3:
+                    {
+                        Console.Clear();
+                        EditContact();
+                        break;
+                    }
+
+                case 4:
+                    {
+                        Console.Clear();
+                        DeleteContact();
+                        break;
+                    }
+
+                case 5:
+                    {
+                        Console.Clear();
+                        int count = 0;
+                        foreach (var obj in _contactList)
+                        {
+                            count++;
+                            Console.WriteLine($"Name : {obj.Name}  P.No. : {obj.PhoneNumber}");
+                            Console.WriteLine($"Email ID : {obj.Email} Description : {obj.Note}");
+                            Console.WriteLine("\n--------------------------------------------------------\n");
+                        }
+
+                        Console.WriteLine($"{count} results found");
+                        Console.WriteLine("\npress enter to continue...");
+                        Console.ReadLine();
+                        break;
+                    }
+
+                case 6:
+                    _flag = true;
+                    break;
+                default:
+                    Console.WriteLine("Incorrect value");
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// StringChoise method is used to choose the choice as per the string data type.
+        /// </summary>
+        /// <param name="userInput">choise string</param>
+        public static void StringChoise(string userInput)
+        {
+            switch (userInput)
+            {
+                case "new contact":
+                    {
+                        Console.Clear();
+                        AddNewContact();
+                        break;
+                    }
+
+                case "search":
+                    {
+                        Console.Clear();
+                        SearchContact();
+                        break;
+                    }
+
+                case "search contact":
+                    {
+                        Console.Clear();
+                        SearchContact();
+                        break;
+                    }
+
+                case "edit":
+                    {
+                        Console.Clear();
+                        EditContact();
+                        break;
+                    }
+
+                case "delete":
+                    {
+                        Console.Clear();
+                        DeleteContact();
+                        break;
+                    }
+
+                case "view all":
+                    {
+                        Console.Clear();
+                        int count = 0;
+                        foreach (var obj in _contactList)
```

Comment:
```suggestion
                        foreach (var contact in _contactList)
```
update variable name